test-packages/e2e-tests/package.json

```diff
@@ -43,6 +44,7 @@
     "json-schema-faker": "^0.5.0-rcv.46",
     "mock-fs": "^5.2.0",
     "moment": "^2.29.4",
+    "nock": "^13.2.9",
```

Comment:
[req] In the e2e test package, `nock` as a mock package should not be used?

Reply:
removed